### PR TITLE
Send an email when a server is marked as installed

### LINKED ISSUE #1202: No email sent when server is marked as installed.
Panel or Daemon: Panel
Version of Panel/Daemon: Panel 0.7 / Wings 0.5.6
Server's OS: Debian 9 virtual machine KVM
Your Computer's OS & Browser: Windows 10 / Chrome-Firefox

Hi guys !
First of all, I'm new here, just discovered your work a few days ago, so thank you for this awesome job !

I've already run through a couple of problems which I was able to solve on my own, but I have still one and I'm not sure if it's a bug or a not implemented feature.

Here's the situation : 
I had trouble sending mail, but figured out it was a server config issue, fixed it, restarted VM to be sure, mails are now sending fine to new users when creating account, and I can see them on my mail server's logs.
But, when I create a server, booting it up or not, the assigned user never receive the mail, despite the install page saying "Server installation in progress, you will receive an email when process is complete". And if I check my mail server's logs, I find no trace of a failed connection, nor a mail passing through.
I also checked the table "jobs" and "failed_logs", 1st is empty, 2nd one contains some errors but the timestamp shows it was during older tests.

So my question is, is this mail supposed to be sent ? I've already do a lot of research in docs/guides/github/google but cannot find any clue on this one.

Anyway, thanks again for your work ! I will try to contribute, at least for langs, and try to lay some eggs :)

### PR DESCRIPTION
**Pull request by stanjg**
Fixes #1202

### CODE REVIEW

**Comment by stanjg** [comment]
Wait, I did not. What happened?

**Comment by DaneEveritt** [comment]
Can you apply this patch to fix the styleCI warnings?

```diff
diff --git a/app/Events/Server/Installed.php b/app/Events/Server/Installed.php
index 96fede48cbfe1d7bfd91956d540283fe25d6420b..a8552c798446a9aa766baf5f09fd7b085775f482 100644
--- a/app/Events/Server/Installed.php
+++ b/app/Events/Server/Installed.php
@@ -2,9 +2,8 @@
 
 namespace Pterodactyl\Events\Server;
 
-use Illuminate\Broadcasting\Channel;
-use Illuminate\Queue\SerializesModels;
 use Pterodactyl\Models\Server;
+use Illuminate\Queue\SerializesModels;
 
 class Installed
 {
@@ -24,5 +23,4 @@ class Installed
     {
         $this->server = $server;
     }
-
 }
diff --git a/app/Http/Controllers/Daemon/ActionController.php b/app/Http/Controllers/Daemon/ActionController.php
index 31fb62c0cbd49375e3dd24c6b5cafbe3d8c2da39..236dc5b24ff3e222f13f6247dc77d8da9f807b9c 100644
--- a/app/Http/Controllers/Daemon/ActionController.php
+++ b/app/Http/Controllers/Daemon/ActionController.php
@@ -4,9 +4,9 @@ namespace Pterodactyl\Http\Controllers\Daemon;
 
 use Cache;
 use Illuminate\Http\Request;
-use Pterodactyl\Events\Server\Installed;
 use Pterodactyl\Models\Node;
 use Pterodactyl\Models\Server;
+use Pterodactyl\Events\Server\Installed;
 use Pterodactyl\Http\Controllers\Controller;
 
 class ActionController extends Controller
diff --git a/app/Notifications/ServerInstalled.php b/app/Notifications/ServerInstalled.php
index 18eda95bd132219e9110c6a4bf6e9c42cd4b05f0..82e318e3ad35c89227da81c2b41b42116dcafb99 100644
--- a/app/Notifications/ServerInstalled.php
+++ b/app/Notifications/ServerInstalled.php
@@ -3,10 +3,10 @@
 namespace Pterodactyl\Notifications;
 
 use Illuminate\Bus\Queueable;
-use Illuminate\Contracts\Queue\ShouldQueue;
+use Pterodactyl\Models\Server;
 use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Notifications\Messages\MailMessage;
-use Pterodactyl\Models\Server;
 
 class ServerInstalled extends Notification implements ShouldQueue
 {
diff --git a/app/Providers/EventServiceProvider.php b/app/Providers/EventServiceProvider.php
index 05622f339667a68ff5304750e95f9391fa3ec5ea..322d7a8649ca9becc503ea8b054b514fbb15d04b 100644
--- a/app/Providers/EventServiceProvider.php
+++ b/app/Providers/EventServiceProvider.php
@@ -2,9 +2,9 @@
 
 namespace Pterodactyl\Providers;
 
-use Illuminate\Foundation\Support\Providers\EventServiceProvider as ServiceProvider;
 use Pterodactyl\Events\Server\Installed;
 use Pterodactyl\Listeners\Server\SendInstalledEmail;
+use Illuminate\Foundation\Support\Providers\EventServiceProvider as ServiceProvider;
 
 class EventServiceProvider extends ServiceProvider
 {
```

File: app/Http/Controllers/Daemon/ActionController.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 use Cache;
 use Illuminate\Http\Request;
+use Pterodactyl\Events\Server\Installed;
```
Comment (DaneEveritt) [inline app/Http/Controllers/Daemon/ActionController.php]: I probably don't do this elsewhere, but can we import this as `ServerInstalledEvent` so when it is used in the code its obvious what we are doing?

File: app/Notifications/ServerInstalled.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->error()
+            ->greeting('Hello ' . $this->server->user . '.')
```
Comment (DaneEveritt) [inline app/Notifications/ServerInstalled.php]: Does this work? Isn't `$this->server->user` just going to give a `User` model?

**Pull request merged**